components/app/dao-proposals/DaoProposalsDropdowns.vue

```diff
@@ -38,7 +42,8 @@ export default {
           type: 'rounds',
           content: `All`,
           id: 0,
-          selected: true,
+          // eslint-disable-next-line no-unneeded-ternary
+          selected: this.filter.round === 0 ? true : false,
```

Comment:
suggestion(blocking): `this.filter.round === 0` resolves to true or false - no need for `? true : false`